### DISCUSSION TITLE
Add "take a break timer"

### DISCUSSION THREAD

**Discussion by GregEisenberg** [remotemobprogramming/mob#173] [Ideas]
One idea my team had after using the tool for a little while is to build in a break timer. None of us are really Go developers so we have not yet tried to create our own PR to add this, but I figured I would throw this out there for community. We may make it a team exercise in the future.  

The Break Timer design does have a bit of trickiness because you have to decide from which time point do you want to start the timer and where does it run.  i.e. if started on the first person's computer when the first Mob Start occurs - what if that person has left the mob by the time the timer pops.  I was thinking about some way to have each mob next/mob start convey info to the next.  Or perhaps just base it on the number of mob next's and not really on elapsed time.  Anyway - let the discussion start.

**Comment by simonharrer** [comment]
Great idea. We had that I think at some point in time as well. But haven’t found a good way to implement it. Perhaps together we can find a sensible way.

Some issues I remember:
- some mob next calls create no commit as nothing changed
- how to account for changes like people joining, leaving, skipping, etc.
- where to store the info of when to have a break and how to distribute that info to everybody collaborating
- what happens if a team needs to work on multiple git repos during a mob session?

**Comment by simonharrer** [comment]
We've added a break timer now. Have a look at the most recent version. :-)

**Reply by Gerrri** [reply]
We are intreseted in a simillar Timer feature too! (We are no-go developers too)
For us it would be great to have a simple Timer (on the Website) that rings (or say something like "take a break") every x Minutes to remind us to take a break (hit the break Button). After x Minutes taking a break the Timer will be started again automatically.

should i open another discussion for this?

**Reply by simonharrer** [reply]
have you had a chance to have a look at timer.mob.sh? It'snot exactly what you want but it might still help you.

**Reply by Gerrri** [reply]
yes i took a look at the timer feature, but its not what we need for our Problem. I dont know if it is a very special wish/idea or others do need the same feature. Our main problem is forgetting to take a break 😄
